modules/ts/misc/run.py

```diff
@@ -850,12 +850,49 @@ def getRunArgs(args):
             path = npath
     return run_args
 
+def moveTests(instance, destination):
+    src = os.path.dirname(instance.tests_dir)
+    # new binaries path
+    newBinPath = os.path.join(destination, "bin")
+
+    try:
+        # copy binaries and CMakeCache.txt to the specified destination
+        shutil.copytree(src, newBinPath)
```

Comment:
This works fine only for Windows-based binaries. On Linux-bases systems library code is stored in `lib` directory, so this movement will not provide standalone tests "package" (also there is RPATH handling problem, so it is not easy to fix).
If you are not interested in Linux-based systems, then you can enable this feature only for Windows.